src/CommonSocket.php

```diff
@@ -113,6 +113,13 @@
      */
     protected $config = null;
 
+    /**
+     * Sasl provider
+     * @var SaslMechanism
+     * @access protected
```

Comment:
I know that we have this everywhere but all tools that generates documentation would simply rely on the attribute/method/constant visibility, so I'd simply drop it